support-workers/src/main/scala/com/gu/support/workers/lambdas/CreatePaymentMethod.scala

```diff
@@ -26,7 +26,7 @@ class CreatePaymentMethod(servicesProvider: ServiceProvider = ServiceProvider)
       .map(paymentMethod =>
         HandlerResult(
           getCreateSalesforceContactState(state, paymentMethod),
-          requestInfo.appendMessage(s"Payment method is ${paymentMethod.toFriendlyString}")
```

Comment:
Did we intend to remove this here?

Reply:
No, this is a mistake. I should not have changed this lambda

Reply:
In fact maybe the new lambda that retrieves an existing payment method should also put this message in requestInfo to keep it consistent

Reply:
rebased out this line 😄